pkgs/development/web/cypress/default.nix

```diff
@@ -0,0 +1,43 @@
+{stdenv, fetchurl, autoPatchelfHook, xorg, gtk2, gnome2, nss, alsaLib, udev, unzip}:
+
+stdenv.mkDerivation rec{
+  version = "3.2.0";
+  name = "cypress-${version}";
```

Comment:
```suggestion
  pname = "cypress";
```

Reply:
@ryantm Sure, I'll make the change as requested. I haven't found any documentation on how to maintain multiple versions of a package. In this case, We might need to move forward with 3.1.5 until We resolve the latest issue in 3.2.0 as reported by @macalinao. Can you point me in the right direction? Thanks.

Reply:
The simplest thing is something like:

```
cypress = callPackage ./development/web/cypress/default.nix { }
cypress_3_1 = callPackage ./development/web/cypress/3.1.nix { }
```
Where `cypress` is the latest version and `cypress_3_1` is the earlier version needed by some specific package. Each would have their own derivation file to avoid breaking eachother.